gateways/adyen/adyen.go

```diff
@@ -159,6 +160,17 @@ func addAdditionalDataFields(
 		response.CvvResultRaw = cvcRaw.(string)
 	}
 
+	// set additional recurring info on response
+	if recurringDetailsReference, isPresent := additionalData["recurring.recurringDetailReference"]; isPresent {
```

Comment:
can we break this in a similar function like RTAU? We can scope it to just recurring detail reference